docroot/modules/humsci/hs_layouts/hs_layouts.module

```diff
@@ -13,8 +13,16 @@ use Drupal\paragraphs\ParagraphInterface;
  */
 function hs_layouts_plugin_filter_layout__layout_builder_alter(&$definitions, &$extra, &$consumer) {
   /** @var \Drupal\Core\Layout\LayoutDefinition $definition */
+  $active_theme = \Drupal::theme()->getActiveTheme()->getName();
```

Comment:
I was checking and there's a more robust way to test if we're in the Dashboard. `$extra` includes a `section_storage` element that will be an instance of `Drupal\dashboard\Plugin\SectionStorage\DashboardSectionStorage` if the current layout builder instance is a dashboard.

Also, having all conditions in a single `if` is a bit confusing. I'd change it to something like this:

```
if ($extra['section_storage'] instanceof DashboardSectionStorage) {
  // Logic for filtering the sections that will be available for the dashboard.
}
else {
  // Logic for keeping only the sections defined in the `hs_layout` module.
}
```

Also:
- Make sure to add `use Drupal\dashboard\Plugin\SectionStorage\DashboardSectionStorage;` in the top of the file.
- Add the dashboard module as a dependency for `hs_layouts` in the `info.yml` file